builds/fabric8-che/plugins/keycloak-plugin-shared/src/main/java/com/redhat/che/keycloak/shared/KeycloakSettings.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.che.keycloak.shared;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.Map;
+
+import java.util.logging.Logger;
```

Comment:
I wouldn't recommend using SLF4J in valve if possible. Because it can produce some strange issues with class loaders. now we package logback in each war too.